### PR TITLE
Update AlphaWallet pods to swift 5.0

### LINKED ISSUE #6751: Update our pods to swift 5.0
(re)-Assign accordingly

### PR DESCRIPTION
**Pull request by eviltofu**
Closes #6751

### CODE REVIEW

**Comment by hboon** [comment]
Can you help remove what seems to be formatting changes first?

**Comment by eviltofu** [comment]
> Can you help remove what seems to be formatting changes first?

Done. @hboon

File: Podfile [hunk 1]
```diff
@@ -100,6 +100,12 @@ post_install do |installer|
       create_symlink_phase.always_out_of_date = "1"
     end
 
+    if target.name.start_with? "AlphaWallet"
+      target.build_configurations.each do |config|       
+        config.build_settings['SWIFT_VERSION'] = '5.0'
```
Comment (hboon) [inline Podfile:105]: Why not set this in the project file instead of overriding it here?

Comment (eviltofu) [reply Podfile:105]: Something overrides the settings. I set the swift language version to 5.0 and running make bootstrap resets it to 4.2. Also git does not register any changes that could be saved. @hboon

Comment (hboon) [reply Podfile:105]: @eviltofu how did you verify this:

> running make bootstrap resets it to 4.2.

Comment (eviltofu) [reply Podfile:105]: I change the Swift Language Version to 5.0 of a Pod. Then I go to the command line and type in `make bootstrap`. After `make bootstrap` is done, I go back to Xcode and check the version number.

Before | After
-|-
before | after

Comment (eviltofu) [reply Podfile]: Updated all AlphaWallet* podspec files to use swift version 5.0 @hboon

**Review by hboon — APPROVED** [review]

**Pull request merged**